src/components/SecurityHeader/SecurityHeader.js

```diff
@@ -327,7 +327,7 @@ export default class SecurityHeader extends Component {
           </div>
         )}
         <ScrollGradient color={theme.inverse02}>
-          <ul className={`${namespace}__popover__list`}>
+          <div className={`${namespace}__popover__list`}>
```

Comment:
We really should try to keep semantic hierarchy. ☝️ 
(Though worth noting that only `ul`/`li` elements can be children of a `ul`)

Reply:
Yeah, I agree, and I wasn't proud of this! 😆Because we want this area to be configurable for consumers to provide notifications or content in the absence of notifications, it no longer made sense to restrict it to a `ul`